### DISCUSSION TITLE
Any way to use client certs for a proxy API?

### DISCUSSION THREAD

**Discussion by ScottChapman** [mockoon/mockoon#1609] [Help]
Our company generally protects APIs with client certs. This, of course, makes doing proxy mocks tricky, and I do not think this is currently supported by Mockoon.

I am following a similar (but I think different?) issue for a feature to support client certs, but I don't know if that would also give this support or not. Or if there is some clever way to do this today?

**Comment by 255kb** [comment]
Indeed, this is not supported and I don't know any workaround. 
We are using http-proxy-middleware and I see that it should be possible to forward the client cert ([here](https://github.com/chimurai/http-proxy-middleware/issues/388) and [here](https://github.com/chimurai/http-proxy-middleware/blob/77f320e4dca8247167471393dcbce2b1e333b7cf/recipes/https.md?plain=1#L18)). 

PR #1604 could help but probably not in its current state.  I'm not sure we can add support for this without a lot of development and testing. 
Also, I'm a bit reluctant to add a feature that wasn't requested much (third time in 7 years).
